AreaSin.java

```diff
@@ -1,13 +1,32 @@
 class AreaSin extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * This program will calculate the area of a triangle, given two side and the contained angle
+  * @author: T. Martins
   */
   
   public void run() {
     
-    // start coding here
-    
+    // declare variables
+    double dblSide1;
+    double dblSide2;
+    double dblAngle; 
+    double dblArea;
+
+    // get values of first side, second side, and contained angle in degrees
+    System.out.println("");
+    dblSide1 = readDouble("Input the value of the first side of the triangle in centimetres: ");
+    dblSide2 = readDouble("Input the value of the second side of the triangle in centimetres: ");
+    dblAngle = readDouble("Input the value of the angle contained between these two sides in degrees: ");
+
+    // convert degrees to radians and calculate area
+    dblAngle = Math.toRadians(dblAngle);
+    dblArea = dblSide1 * dblSide2 * Math.cos(dblAngle) / 2;
```

Comment:
I used Math.cos(dblAngle) for some incomprehensible reason, and since I didn't use Math.sin(dblAngle) the answer is incorrect.